runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1511,53 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version %1$u.%2$u of class=%4$.*3$s, the maximum supported major version is %5$u.0; offset=%6$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_3=6
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_4=62
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_5=0
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_6=61
+J9NLS_CFR_ERR_MAJOR_VERSION2.explanation=The major version in the classfile is unsupported on the VM rejecting the classfile.
+J9NLS_CFR_ERR_MAJOR_VERSION2.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MAJOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+# END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MINOR_VERSION2=bad minor version of class %2$.*1$s. Got minor version .%3$u, but .0 is the only valid minor version for the major version %4$u; offset=%5$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_3=1
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_4=63
```

Comment:
Suggest you use consistent examples; above the maximum major version is 61 - it should be the same here.